pkg/apis/fleet.cattle.io/v1alpha1/gitrepo_types.go

```diff
@@ -240,16 +240,21 @@ type GitRepoDisplay struct {
 // GitRepoResource contains metadata about the resources of a bundle.
 type GitRepoResource struct {
 	// APIVersion is the API version of the resource.
+	// +nullable
 	APIVersion string `json:"apiVersion,omitempty"`
 	// Kind is the k8s kind of the resource.
+	// +nullable
 	Kind string `json:"kind,omitempty"`
 	// Type is the type of the resource, e.g. "apiextensions.k8s.io.customresourcedefinition" or "configmap".
 	Type string `json:"type,omitempty"`
```

Comment:
I am surprised to see that many other fields would be nullable but not this one :thinking:

Reply:
<img width="764" alt="Screenshot 2024-07-04 at 18 40 44" src="https://github.com/rancher/fleet/assets/393653/402e2929-efe4-4785-8a5d-61b82711f68e">

Clearly indicated by line 5924 of the patch ... 😛

Reply:
Ah, sorry. I meant https://github.com/rancher/fleet/compare/v0.9.1-rc.3...v0.10.0-rc.19#diff-fe6024d9cdd994ea2d530051f203572ee902f5af2936f095b2bfd4f81f4aca26L5312

(this is so confusing to diff)

Reply:
Fair enough.